scex-util/src/main/scala/com/avsystem/scex/util/function/NetFunctions.java

```diff
@@ -116,8 +116,8 @@ public static boolean isIps(final String ips) {
         }
 
         if (ips != null) {
-            for (String ip : ips.replaceAll(",\\s+", ",").split(",")) {
-                if (!isIp(ip)) {
+            for (String ip : ips.split(",")) {
```

Comment:
Just minor nitpick: `","` is pointlessly treated as regex, if you used a char `','` then you could avoid that.